### ISSUE TITLE
Add the expected date, when the app is going to be active

### ISSUE THREAD

**Issue by hartsberger** [nextcloud/suspicious_login#348]
Hi, 

it would be nice if you could **add the expected date, when the app is going to be active** in the admin settings. (When the 60 days are going to be reached)

This would tell the admin the date when she should have a look on the security settings again. 

## Suggestion of the message (bold): 
Ein Kriterienmodell wurde bis jetzt noch nicht erstellt. Dies bedeutet, dass die Anwendung höchstwahrscheinlich erst kürzlich aktiviert wurde. Um ein Kriterienmodell erstellen zu können, wird eine verlässliche Datenbasis benötigt. Aus diesem Grund wartet die Anwendung, bis Anmeldungen von mindestens 60 Tagen **(voraussichtlich am 01.01.2021)** erfasst wurden. 

Many thanks

**Comment by ChristophWurst** [comment]
sure :) are you a developer? this seems like a good first issue to tackle :)

**Comment by hartsberger** [comment]
Indeed, and I would love to use this case as my first developer-contribution :heart_eyes:  :muscle: 
But unfortunately I can't afford the time the next few days/weeks. 

So if anyone else wants to pick this issue, feel free :slightly_smiling_face: 
Otherwise, I will try my best to work on this alter on.

**Comment by ChristophWurst** [comment]
Sure, take your time :v:
